cmd/wallet-js-sdk/src/oidc/presentation/openid4vp.js

```diff
@@ -175,36 +179,24 @@ export class OpenID4VP {
     }
 
     const header = new Object();
-    Object.defineProperty(header, "alg", {
-      value: alg,
-    });
-    Object.defineProperty(header, "kid", {
-      value: kid,
-    });
-    Object.defineProperty(header, "typ", {
-      value: "JWT",
-    });
+    header["alg"] = alg;
+    header["kid"] = kid;
+    header["typ"] = "JWT";
```

Comment:
:)

Reply:
`const header = new Object();`

can also be written as 

`const header = {}`

Reply:
Yep, the `defineProperty` syntax did not allow me to stringify the objects